samples/nrf9160/modem_shell/Kconfig

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "Nordic Modem Shell config"
+
+config MOSH_IPERF3
+	bool  "Enable iperf3 support"
+	default y
+	select NRF_IPERF3
+
+config MOSH_SOCK
+	bool  "Enable socket tool support"
+	default y
+
+config MOSH_PING
+	bool  "Enable ICMP ping support"
+	default y
+
+config MOSH_LINK
+	bool  "Enable link control support"
+	depends on LTE_LINK_CONTROL
+	depends on SETTINGS
+	default y
+
+config MOSH_CURL
+	bool  "Enable CURL"
+	default y
+	select NRF_CURL
+
+config MOSH_GNSS
+	bool  "Enable GNSS support"
+	default y
+
+config MOSH_PPP
+	bool  "Enable PPP support"
+
+config MOSH_SMS
+	bool  "Enable SMS support"
+	default y
+
+config MOSH_FOTA
+	bool "Enable FOTA support"
+	default y
+
+if MOSH_LINK
+menu "MoSH link command selections"
+
+config MOSH_LINK_SETT_NORMAL_MODE_AT_CMD_STR_LEN
+	int "configure string length for custom AT cmds stored in settings and used when going to normal mode"
+	default 127
+
+endmenu
+endif #MOSH_LINK
+
+choice MOSH_GNSS_API
+	bool "Used GNSS API"
```

Comment:
Perhaps remove `Used`.

![image](https://user-images.githubusercontent.com/9802583/120384814-ef8b7600-c326-11eb-9dfe-31bcc7c124ab.png)

vs.

![image](https://user-images.githubusercontent.com/9802583/120384706-ccf95d00-c326-11eb-9d91-7a46919ab009.png)

This is just a proposal, feel free to keep existing text if you feel that looks nicer.

Reply:
Yep, removed.

Reply:
Removed